wayback/_client.py

```diff
@@ -336,6 +337,8 @@ def send(self, *args, **kwargs):
             try:
                 result = super().send(*args, **kwargs)
                 if retries >= maximum or not self.should_retry(result):
+                    if result.status_code == 429:
+                        raise RateLimitError(result)
```

Comment:
Worth noting here: we raise the error *after* exhausting retries. It’s probably debatable whether that’s correct; I figured that, since the retry wait might be long enough to automatically overcome the error, this was OK.